src/anyvlm/schemas/vlm.py

```diff
@@ -1,9 +1,154 @@
 """Schemas relating to VLM API."""
 
-from pydantic import BaseModel
+from typing import ClassVar, Literal, Self
+
+from pydantic import BaseModel, Field, model_validator
+
+from anyvlm.utils.types import Zygosity
+
+# ruff: noqa: N815 (allows camelCase vars instead of snake_case to align with expected VLM protocol response)
+
+RESULT_ENTITY_TYPE = "genomicVariant"
+
+
+class HandoverType(BaseModel):
+    """The type of handover the parent `BeaconHandover` represents."""
+
+    id: str = Field(
+        default="gregor", description="Node-specific identifier"
+    )  # TODO: enable configuration of this field. See Issue #27.
+    label: str = Field(
+        default="GREGoR AnVIL browser", description="Node-specific label"
+    )  # TODO: enable configuration of this field. See Issue #27.
+
+
+class BeaconHandover(BaseModel):
+    """Describes how users can get more information about the results provided in the parent `VlmResponse`"""
+
+    handoverType: HandoverType = HandoverType()
+    url: str = Field(
+        default="https://anvil.terra.bio/#workspaces?filter=GREGoR",  # TODO: enable configuration of this field. See Issue #27.
+        description="A url which directs users to more detailed information about the results tabulated by the API (ideally human-readable)",
+    )
+
+
+class ReturnedSchema(BaseModel):
+    """Fixed [Beacon Schema](https://github.com/ga4gh-beacon/beacon-v2/blob/c6558bf2e6494df3905f7b2df66e903dfe509500/framework/json/common/beaconCommonComponents.json#L241)"""
+
+    entityType: str = Field(
+        default=RESULT_ENTITY_TYPE,
+        description=f"The type of entity this response describes. Must always be set to '{RESULT_ENTITY_TYPE}'",
+    )
+    schema_: str = Field(
+        default="ga4gh-beacon-variant-v2.0.0",
+        # Alias is required because 'schema' is reserved by Pydantic's BaseModel class,
+        # But VLM expects a field named 'schema'
+        alias="schema",
+    )
+
+    model_config = {"populate_by_name": True}
```

Comment:
I think we should be using ConfigDict (will need to update import) for pydantic v2

```suggestion
    model_config = ConfigDict(populate_by_name=True)
```